tests/integration/mixer/k8s/k8s_test.go

```diff
@@ -1,78 +0,0 @@
-// Copyright 2019 Istio Authors
```

Comment:
This test was removed as it was just smoke testing mixer without Galley. Since this is now the only way to deploy mixer, and is covered by every other mixer tests, its not needed